docs/source/nextflow-workflow.rst

```diff
@@ -135,6 +135,292 @@ Example ``sample_sheet.csv``
+
+.. code-block:: bash
+    
+    conda activate autometa-nf
+
+Run the following code to launch Autometa:
+
+.. code-block:: bash
+
+    nf-core launch KwanLab/Autometa
+
+.. note::
+
+    You may want to note where you have saved your input sample sheet prior to running the launch command. 
+    It is much easier (and less error prone) to copy/paste the sample sheet file path when specifying the input (We'll get to this later in :ref`quickstart-menu-4`).
+
+You will now use the arrow keys to move up and down between your options and hit your "Enter" or "Return" key to make your choice.
+
+**``KwanLab/Autometa`` parameter settings:**
+
+#. :ref:`quickstart-menu-1`
+#. :ref:`quickstart-menu-2`
+#. :ref:`quickstart-menu-3`
+#. :ref:`quickstart-menu-4`
+#. :ref:`quickstart-menu-5`
+#. :ref:`quickstart-menu-6`
+#. :ref:`quickstart-menu-7`
+#. :ref:`quickstart-menu-8`
+
+
+.. _quickstart-menu-1:
+
+**Choose a version**
+
+The double, right-handed arrows should already indicate "2.0.0". This is the latest version of Autometa. 
+The latest version of the tool will always be at the top of the list with older versions descending below. 
+To select the latest version, ensure that the double, right-handed arrows are next to "2.0.0", then hit "Enter".
+
+.. image:: ../img/Menu1.png
+
+.. _quickstart-menu-2:
+
+**Choose 'launch' method**
+
+Pick the ``Command line`` option.
+
+.. note::
+
+    Unless you've done some fancy server networking (i.e. tunneling and port-forwarding), 
+    or are using Autometa locally, ``Command line`` is your `only` option.
+
+.. image:: ../img/Menu2.png
+
+.. _quickstart-menu-3:
+
+**General nextflow parameters**
+
+If you are using a scheduler (Slurm in this example) the only thing you'll need to change is the ``-profile`` option. 
+If you are not using a scheduler, you may skip this step.
+
+.. image:: ../img/Menu3.png
+
+.. _quickstart-menu-4:
+
+**Input and Output**
+
+Now we need to give Autometa the full paths to our input sample sheet, output results folder 
+and output logs folder (aka where trace files are stored). 
+
+.. note::
+
+    A new folder, named by its respective sample value, will be created within the output results folder for 
+    each metagenome listed in the sample sheet.
+
+.. image:: ../img/Menu4.png
+
+.. _quickstart-menu-5:
+
+**Binning parameters**
+
+If you're not sure what you're doing I would recommend only changing ``length_cutoff``. 
+The default cutoff is 3000bp, which means that any contigs/scaffolds smaller than 3000bp will not be considered for binning. 
+
+.. note::
+
+    This cutoff will depend on how good your assembly is: e.g. if your N50 is 1200 bp, I would choose a cutoff of 1000. 
+    If your N50 is more along the lines of 5000, I would leave the cutoff at the default 3000. I would strongly recommend
+    against choosing a number below 900 here. In the example below, I have chosen a cutoff of 1000bp as my assembly was 
+    not particularly great.
```

Comment:
What was the N50 for the assembly you mention? This may be helpful to the reader

> In the example below, I have chosen a cutoff of 1000bp as my assembly was 
    not particularly great.